### PR TITLE
Docs: Fix broken links

### PR DESCRIPTION
**Pull request by AlecGard**
Links to Ethereum and Corda workspace overview on https://trufflesuite.com/docs/ganache/quickstart are currently directing to a 404 page.

This commit fixes these links to direct to the correct pages.

### CODE REVIEW

**Comment by davidmurdoch** [comment]
Thanks for opening this @AlecGard, I'm not able to reproduce this. Can you take a screenshot of the specific links on the site that are 404ing for you?

**Review by davidmurdoch — APPROVED** [review]

**Comment by AlecGard** [comment]
Thanks for approving @davidmurdoch - I don't have write access so I'll leave it to you to merge!

**Comment by AlecGard** [comment]
Bump: These links are still broken and this PR to fix is approved, but I don't have write access. Please can a maintainer merge this PR

**Pull request merged**
